sdk/resources/mgmt/src/main/java/com/azure/management/resources/implementation/ResourceManager.java

```diff
@@ -57,24 +55,19 @@
      * @param credential the credentials to use
      * @return the ResourceManager instance
      */
-    public static ResourceManager.Authenticated authenticate(AzureTokenCredential credential) {
-        return new AuthenticatedImpl(new RestClientBuilder()
-                .withBaseUrl(credential.getEnvironment(), AzureEnvironment.Endpoint.RESOURCE_MANAGER)
-                .withCredential(credential)
-                .withSerializerAdapter(new AzureJacksonAdapter())
-                .withPolicy(new ProviderRegistrationPolicy(credential))
-                .withPolicy(new ResourceManagerThrottlingPolicy())
-                .buildClient());
+    public static ResourceManager.Authenticated authenticate(TokenCredential credential) {
+        return new AuthenticatedImpl(HttpPipelineProvider.buildHttpPipeline(credential),
+            new AzureProfile(AzureEnvironment.AZURE));
```

Comment:
Most of user will use this simple `authenticate`, I think maybe `AzureProfile` is needed. Not sure whether they need to set `tenantId` or not.

Reply:
Agree with this, I will change it to apply `AzureProfile` together